### DISCUSSION TITLE
The use of Github App results in "403 Resource not accessible by integration"

### DISCUSSION THREAD

**Discussion by taneli-kantomaa** [actions/actions-runner-controller#3742] [Questions]
Hi, I am trying to use Github App as the authentication method, but currently this results in an error "403 Resource not accessible by integration". The PAT authentication method works without problems. Here are the details:

--Setup
aks cluster: v1.28.9
actions-runner-controller helm chart version: 0.23.7
controller-image version: 0.27.6

--Description
I am trying to configure actions-runner-controller (with a Helm chart) to run Github actions from a repository (one single repository) in my Github organization (for this Organization I have an owner role). Everything works fine when I configure a PAT token, but when I try to replace the PAT token with a Github App (under the organization, not by using my personal account) I run into the following error: "403 Resource not accessible by integration" . As per my understanding the Github App configuration in Github as well as the values.yaml file for the helm chart installation have been configured correctly:

Github App (which is owned by the organization) has the following permissions:
-Repository Permissions: Actions (read + write), Checks (read + write), Contents (read + write), Metadata (read), Workflows (read + write)
-Organization Permissions: Self-hosted runners (Read + write)

Values file

githubApp:
enabled: true
id: ${GITHUB_APP_ID}
installationId: ${GITHUB_APP_INSTALLATION_ID}
privateKey: ${GITHUB_APP_PRIVATE_KEY}

Any help on this issue would be highly appreciated. Thank you.

**Comment by hungphaminnomizetech** [comment]
Hi @taneli-kantomaa, First you need to make sure the GitHub App is installed and interacts with the repository. And if you have any update permission of GitHub App. Please reinstall GitHub App
